docs/en/Running-Alluxio-Fault-Tolerant.md

```diff
@@ -152,3 +152,30 @@ Alluxio leader master.
 ```
 hadoop fs -ls alluxio:///directory
 ```
+
+### Automatic Fail Over
+
+To test automatic fail over, ssh into current Alluxio master leader, and find process ID of
+the `AlluxioMaster` process with:
+
+```bash
+$ jps | grep AlluxioMaster
+```
+
+Then kill the leader with:
+
+```bash
+$ kill -9 <leader pid found via the above command>
+```
+
+Then you can run the zookeeper client shell, and see the leader with the command:
```

Comment:
We should recommend running `alluxio fs leader` instead to determine the leader master address. The way we use zookeeper is an implementation detail and might change in the future

Reply:
Updated.